### PR TITLE
Linking and Formatting

### PR DESCRIPTION
**Pull request by mbround18**
Contributions:

- Updated linking in sub files
- Fixed formatting, please use linting for consistency. 
- Added `.editorconfig`

### CODE REVIEW

**Comment by mbround18** [comment]
In the latest commit, I added a segment about PaperMC for 1.16+.

**Comment by JustDoom** [comment]
Sorry but one of the other pull requests changed a file and won't let me merge, it's advanced jar picking

**Comment by mbround18** [comment]
@JustDoom fixed

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on: push
```
Comment (Silverarmor) [inline .github/workflows/lint.yml:3]: This should probably be
`on: [push. pull_request]`

File: tutorials/en_us/Advanced JAR picking.md [hunk 2]
```diff
@@ -1,70 +1,51 @@
...
-5. Tuinity
-#
+If you want to go a little more in-depth with JAR picking, you've come to the right place.
+Most popular Spigot/Bukkit JARs are :
 
-1. PaperSpigot - Most popular JAR for Spigot/Bukkit, quite fast and stable, they roll out small patches at least every 2 days or so, good choice if you're just
-                 starting with your MC server and don't have much past experience because it's fast and simple.
+1. [PaperMC](https://papermc.io/) - Most popular JAR for Spigot/Bukkit, quite fast and stable, they roll out small patches at least every 2 days or so, good choice if you're just
+   starting with your MC server and don't have much past experience because it's fast and simple.
+1. [Spigot](https://www.spigotmc.org/) - Original fork of Bukkit, improved performance vs CraftBukkit, still widely used, seems to be harder for newbies to download because it links to Jenkins,
```
Comment (Silverarmor) [inline tutorials/en_us/Advanced JAR picking.md:8]: Why use a numbered list here?
Also they probably shouldn't all be `1.`
Imo change to unordered list - using `-` dashes.

File: tutorials/en_us/Raspberry_pi_mcserver.md [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
 You may want to install plugins like WorldEdit or WorldGuard...
 And this is the end of the tutorial.
 
-# FAQ
+## FAQ
+
 Here are some basic answers.
 
-## How to change RAM?
+### How to change RAM?
```
Comment (Silverarmor) [inline tutorials/en_us/Raspberry_pi_mcserver.md:33]: "change RAM"?

Comment (Silverarmor) [reply tutorials/en_us/Raspberry_pi_mcserver.md:33]: possibly "How to change allocated RAM" or similar

**Review by Silverarmor — CHANGES_REQUESTED** [review]

**Comment by JustDoom** [comment]
so do i accept this?

**Comment by mbround18** [comment]
let me make some updates and fix merge conflicts

**Comment by mbround18** [comment]
@JustDoom updated

**Review by Silverarmor — APPROVED** [review]

**Comment by Kruemmelspalter** [comment]
@mbround18 please resolve the conflicts

**Comment by mbround18** [comment]
lol im getting tired of resolving the same conflicts repetitively

**Comment by JustDoom** [comment]
sorry about that

**Comment by mbround18** [comment]
I removed my changes in those files to avoid conflicts should other branches get merged.

**Pull request merged**